### PR TITLE
2.1.0 release

### PR DESCRIPTION
**Pull request by camillebaldock**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "heroku-pg",
   "description": "Heroku CLI plugin to manage Postgres.",
-  "version": "2.0.26",
+  "version": "2.0.27",
```
Comment (msakrejda) [inline package.json]: This is for the creds changes, yeah? Shouldn't that be a `2.1.0`?

**Review by msakrejda — APPROVED** [review]
Minor suggestion, but +1

File: package.json (hunk 1 above)
Comment (camillebaldock) [reply package.json]: it doesn't break any of the existing behaviour but it does add a lot of new commands so yes agreed, i'm updating now

**Comment by camillebaldock** [comment]
@dickeyxxx hi! what is the process for releasing updated npm packages for a new heroku-pg version?

**Comment by camillebaldock** [comment]
Has been released

**Pull request closed without merging**